app/workers/list_sync/sync_worker.rb

```diff
@@ -0,0 +1,25 @@
+module ListSync
+  class SyncWorker
+    include Sidekiq::Worker
+    sidekiq_options retry: false, queue: 'sync'
+
+    def perform(linked_account_id, user_id)
+      user = User.find(user_id)
+      linked_account = LinkedAccount.find(linked_account_id)
+      entries = user.library_entries
+      entries.map do |le|
+        LibraryEntryLog.create_for(:update, le, linked_account)
+      end
+      pending_logs = LibraryEntryLog.for_entry(library_entry).pending
```

Comment:
same with this one, where is `library_entry` defined? Also because this one has multiple entries would this actually work?

Possible logical error here?